internal/actions/pr.go

```diff
@@ -135,11 +141,14 @@ func CreatePullRequest(ctx context.Context, repo *git.Repo, client *gh.Client, o
 
 	pull, didCreatePR, err := getOrCreatePR(ctx, client, repoMeta, getOrCreatePROpts{
 		baseRefName: prBaseBranch,
-		headRefName: currentBranch,
+		headRefName: opts.BranchName,
 		title:       opts.Title,
 		body:        opts.Body,
 		draft:       opts.Draft,
 	})
+	if err != nil {
```

Comment:
🙀 nice catch

Reply:
We should add go linting to our CI. It can catch this kind of thing (unchecked `err`).